pkg/render/logstorage/dashboards/dashboards.go

```diff
@@ -49,6 +50,18 @@ var (
 	PolicyName               = networkpolicy.TigeraComponentPolicyPrefix + Name
 )
 
+// GetJobName makes a unique job name per operator version.
+// For dev images it takes the first 32 chars.
+func GetJobName() string {
```

Comment:
If we're switching to this, we should stop exporting the `Name` variable so that nobody accidentally uses the wrong value.

Reply:
If `Name` isn't actually used except for within this function, we should probably just scope it to this function to be extra sure (i.e., remove the package level variable)